### DISCUSSION TITLE
solution error install seb 3.31 on win 10 x64

### DISCUSSION THREAD

**Discussion by smasaae** [SafeExamBrowser/seb-win-refactoring#267] [General]
when facing error install seb 3.31 on win 10, do following trick
1. install VC_redist.x64.14.24.28127.4 if newer version installed, then ignored error
2. install MicrosoftEdgeEnterpriseX64.msi >> or do updated MS EDGE browser to latest version
3. install NET. Framework 4.72-kb4054530-x86-x64-allos-enu.exe if newer version installed, then ignored error
4. connect to internet
5. open Winkey + R, type wsreset.exe then enter after 10 second or close it
6. open cmd as administrator >> type netsh winhhtp reset proxy
7. install SEB_3.3.1.388_SetupBundle.exe
8. voila

**Comment by dbuechel** [comment]
The exact runtime dependencies are listed in the [readme](https://github.com/SafeExamBrowser/seb-win-refactoring#readme) as well as the [release notes](https://github.com/SafeExamBrowser/seb-win-refactoring/releases/tag/3.3.1) here on GitHub. There is no need to install the enterprise edition of Edge, for example. Furthermore, the steps you provide above appear rather random. It does not really make any sense that one would need to reset / clear the Windows Store (wsreset.exe) or reset the WinHTTP proxy in order to be able to install SEB.

Could you please provide the exact error message you encountered?

**Reply by smasaae** [reply]
on error 0x80040801, 0x80072EFD

**Comment by smasaae** [comment]
![IMG20211125110030](https://user-images.githubusercontent.com/94957015/143582328-4fc18a94-cc9f-4328-9d52-c80a99bdbeda.jpg)

**Reply by dbuechel** [reply]
Thanks for the information. Could you please also post the log file of the setup bundle?
